src/cleanlab_tlm/utils/tlm_calibrated.py

```diff
@@ -21,9 +21,95 @@
 
 if TYPE_CHECKING:
     from collections.abc import Sequence
-
     from cleanlab_tlm.internal.types import TLMQualityPreset
 
+def _get_skops():
+    """Lazy import for skops to avoid unnecessary dependency."""
+    try:
+        import skops.io
+        return skops.io
+    except ImportError:
+        raise ImportError(
+            "The skops package is required for model serialization. "
+            "Please install it with: pip install skops"
+        )
+
+def save_tlm_calibrated_state(model: 'TLMCalibrated', custom_eval_options: Dict[str, Any], filename: str) -> None:
+    """Save fitted TLMCalibrated model state to file.
+    
+    Args:
+        model: A fitted TLMCalibrated model instance
+        custom_eval_options: Dictionary containing custom evaluation options
+        filename: Path where the model state will be saved
+    
+    Raises:
+        sklearn.exceptions.NotFittedError: If the model has not been fitted
+        ImportError: If skops package is not installed
+    """
+    try:
+        from sklearn.utils.validation import check_is_fitted
+    except ImportError:
+        raise ImportError(
+            "Cannot import scikit-learn which is required to use TLMCalibrated. "
+            "Please install it using `pip install scikit-learn` and try again."
+        )
+    
+    # Verify model is fitted
+    check_is_fitted(model._rf_model)
+    
+    # Capture essential state
+    state = {
+        'options': custom_eval_options,
+        'rf_state': {attr: getattr(model._rf_model, attr, None) 
+                    for attr in ['n_features_in_', 'n_outputs_', 'estimators_', 'monotonic_cst_']},
+        'train_scores': getattr(model, '_train_scores', None),
+        'train_labels': getattr(model, '_train_labels', None)
```

Comment:
I'm a little confused where these two are coming from, they are not attributes in the `TLMCalibrated` object?